hackerank/js-practice/angry-professor.js

```diff
@@ -2,18 +2,23 @@
 
 // Complete the angryProfessor function below.
 function angryProfessor(studentsThreshold, arrivalTime) {
+  const canceledClass = "YES";
```

Comment:
You can define the constant outside the scope of the function and you could reuse it on lines :23 :24